build/generic/web/viewer.html

```diff
@@ -26,23 +26,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <meta name="google" content="notranslate">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="<?= $assetPath ?>/viewer.css">
+
+    <script>
+       window.workerPath = <?= $workerPath ?>;
+    </script>
+    <script src="<?= $scripts[0] ?>"></script>
```

Comment:
Could you wrap the lines with `<?= ?>` in `<!--#expand  -->` so that they don't cause syntax errors when testing with `gulp server`?

Example:

```
<!--#expand <script src="<?= $scripts[0] ?>"></script> -->
```